docs/config/README.md

```diff
@@ -285,6 +286,35 @@ it would have been `nixpkgs/pkgs`.
 truncation_length = 8
 ```
 
+## Environment Variable
+
+The `env_var` module displays the current value of a selected environment variable.
+The module will be shown only if any of the following conditions are met:
+
+- The environment variable named `variable` is defined
```

Comment:
I think this may be misleading.
My first impression is that it would be required to have `$VARIABLE` be set.
Maybe instead it should be something along these lines:

```suggestion
- The `variable` configuration option matches an existing environment variable
```

Does this sound clear to folks?